keras_nlp/samplers/sampler.py

```diff
@@ -86,31 +88,31 @@ def __call__(
         self,
         next,
         prompt,
-        state=None,
+        cache=None,
         index=0,
         mask=None,
         end_token_id=None,
+        hidden_states=None,
```

Comment:
What will happen if the hidden state is `None` for the contrastive sampler?

Reply:
it's going to crash - -", thx for catching it!